recipes/blosc-hdf5-plugin/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "blosc-hdf5-plugin" %}
+{% set version = "1.0.0" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+  - git_url: https://github.com/Blosc/hdf5-blosc 
+    git_rev: v{{ version }}
```

Comment:
Please use a tarball, not a repo. See https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

Reply:
OK. The docs say, one should also declare the filename, but this is obsolete (at least upstream). Just wanted to mention it.

Reply:
Yeah, feel free to update the docs in that regard. Thanks!

Reply:
https://github.com/conda-forge/conda-forge.github.io/pull/709